next/src/components/common/NavBar/NavMenu/NavMenuContent.tsx

```diff
@@ -86,13 +98,24 @@ const NavMenuContent = ({ colCount, sections, colorStyle }: NavMenuContentProps)
           })}
           {/* eslint-enable react/no-array-index-key */}
         </ul>
+        <HorizontalDivider />
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+        <div
+          className="border-b p-8"
+          // Together with onCLick in Viewport, it closes the menu on click outside of container area
+          onClick={(event) => event.stopPropagation()}
+        >
+          <NavMenuLink
+            variant="goToCategoryLink"
+            href={parentHref}
+            ariaLabel={t('NavMenuContent.aria.goToCategory', {
+              category: parentLabel,
+            })}
+          >
+            {t('NavMenuContent.goToCategory')}
+          </NavMenuLink>
+        </div>
```

Comment:
Please make this consistent with OLO as much as possible :) 

<img width="1766" height="482" alt="Image" src="https://github.com/user-attachments/assets/d0305aed-9c1d-4f61-9bea-8e415884e881" />